### ISSUE TITLE
Status of the project?

### ISSUE THREAD

**Issue by nguyenhothanhtam0709** [wbraswell/rperl#117]
I see that the last commit was from 2021. Is this project deprecated?

**Comment by wbraswell** [comment]
Not at all!  We are currently in the process of refactoring the data type system into its own Perl::Types distribution:
https://gitlab.com/perl-types/perl-types

**Comment by wbraswell** [comment]
@nguyenhothanhtam0709 
Are you a Perl programmer?

**Comment by nguyenhothanhtam0709** [comment]
@wbraswell I'm just a Perl newbie.

**Comment by bluet** [comment]
@wbraswell will there be a mirror repo on github?
